src/components/Thought.js

```diff
@@ -108,6 +110,9 @@ export const Thought = connect(({ cursor, cursorBeforeEdit, expanded, expandedCo
           document.getSelection().removeAllRanges()
         })
       }
+      else {
+        saveCurrentCursorOffset()
+      }
```

Comment:
Can you explain why this is done only on desktop here?

Reply:
do we need this on mobile? I reckoned not, since the cursor would go wherever the user taps(since the keyboard is hidden anyways, I see no real value in terms of typing continuation)